docker-compose.development.override.yml

```diff
@@ -0,0 +1,10 @@
+version: '2'
+
+services:
+  ms: &ms
+    build: .
+    environment:
+      DATABASE_URL: postgres://postgres@db:5432/msf_dev
+
+    volumes:
+      - .:/usr/src/app
```

Comment:
we should also add `~/.msf[1-9]` as a volume here and in the other compose file (don't know the current version of the folder)